databases/functions/userFollowUser/user-block.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The eslint disable comment is unnecessary here. The `Model` variable from sequelize is being used, so there's no need to disable the linter for this line.
```diff
- // eslint-disable-next-line no-unused-vars
  const {Model} = require('sequelize');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->